dotorg/dotorg-topic-updater.user.js

```diff
@@ -63,5 +63,15 @@
     $( function() {
         obj.init();
     } );
-})( jQuery, {} );
 
+    // Identify bug tickets
+    let bugTicket = $( document ).text().search( /internal bug ticket reference | ticket number | bug report | bug ticket/i );
```

Comment:
This is very broad - it searches the _entire page contents_. Can we narrow down the selector on this some?